config/interfaces.go

```diff
@@ -0,0 +1,15 @@
+package config
+
+// InterfacesConfig configuration for blocky's interfaces (REST, DoH, ...)
+type InterfacesConfig struct {
+	Rest RESTConfig `yaml:"rest"`
```

Comment:
The option name is a bit misleading as almost all http endpoints could be considered as rest endpoints.
I would suggest `control-api` as name since this option only enables/disables the blocky API.